apollo-ios/Sources/Apollo/ApolloClient.swift

```diff
@@ -130,13 +134,21 @@ public final class ApolloClient: ApolloClientProtocol, Sendable {
 
   public func fetch<Query: GraphQLQuery>(
     query: Query,
-    cachePolicy: CachePolicy? = nil
+    cachePolicy: CachePolicy? = nil,
   ) throws -> AsyncThrowingStream<GraphQLResult<Query.Data>, any Error>
   where Query.ResponseFormat == IncrementalDeferredResponseFormat {
-    return try self.networkTransport.send(
-      query: query,
-      cachePolicy: cachePolicy ?? self.defaultCachePolicy
-    )
+    return try sendQuery(query: query, cachePolicy: cachePolicy, requestConfiguration: requestConfiguration)
+  }
+  private func sendQuery<Query: GraphQLQuery>(
+    query: Query,
+    cachePolicy: CachePolicy?,
```

Comment:
> What's meant to be on the next line after the new `,`?

Same here..

Reply:
This is fixed in next PR. Disregard here.